### PR TITLE
es6 refactoring

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @inikulin @churkin

### CODE REVIEW

File: src/client/sandbox/code-instrumentation/properties/index.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
             },
 
             href: {
-                condition: (el) => {
+                condition: el => {
```
Comment (inikulin) [inline src/client/sandbox/code-instrumentation/properties/index.js]: I wonder if latest eslint has linting options for this. The older version haven't. But AFAIK there was a proposal for this. Can you take a look, please and add eslint rule in context of this PR?

**Comment by inikulin** [comment]
requires rebase now

File: .eslintrc [hunk 2]
```diff
@@ -84,9 +84,14 @@
     "new-cap": [2, { "capIsNew": false }],
     "space-before-function-paren": [2, "always"],
     "no-use-before-define" : [2, "nofunc"],
-    "handle-callback-err": 0
+    "handle-callback-err": 0,
+    "babel/arrow-parens": [2, "as-needed"],
```
Comment (inikulin) [inline .eslintrc]: All this new rules and plugin declaration should be in `src/.eslintrc`

**Comment by LavrovArtem** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
\cc @churkin

**Comment by LavrovArtem** [comment]
ping @churkin

**Comment by churkin** [comment]
lgtm

**Pull request merged**